packages/@ama-sdk/schematics/README.md

```diff
@@ -125,11 +130,11 @@ yarn schematics @ama-sdk/schematics:typescript-core --spec-path ./swagger-spec.y
 
 It is possible to configure the SDK code generation by passing parameters to the generator command line to override the default configuration values.
 The available parameters are:
-* `--spec-path`: Path to the swagger specification used to generate the SDK
-* `--spec-config-path`: Path to the spec generation configuration
-* `--global-property`: Comma separated string of options to give to the openapi-generator-cli
-* `--output-path`: Output path for the generated SDK
-* `--generator-custom-path`: Path to a custom generator
+- `--spec-path`: Path to the swagger specification used to generate the SDK
```

Comment:
sometimes we use `*` sometimes `-` 
in the main readme you did the opposite change why ?

Reply:
it is the linter auto-fix, I guest it reserves `*` for table and `-` for list